site_root/input/resources/profiles/StructureDefinition-kfdrc-phenotype-no-phi.json

```diff
@@ -0,0 +1,113 @@
+{
+  "resourceType": "StructureDefinition",
+  "id": "kfdrc-phenotype-no-phi",
+  "url": "http://fhir.kids-first.io/StructureDefinition/kfdrc-phenotype-no-phi",
+  "version": "0.1.0",
+  "name": "kfdrc-phenotype-no-phi",
+  "title": "Kids First DRC Phenotype without Protected Health Information",
+  "status": "draft",
+  "fhirVersion": "4.0.0",
+  "kind": "resource",
+  "abstract": false,
+  "type": "Observation",
+  "baseDefinition": "http://fhir.kids-first.io/StructureDefinition/kfdrc-phenotype",
+  "derivation": "constraint",
+  "differential": {
+    "element": [
+      {
+        "id": "Observation",
+        "path": "Observation"
+      },
+      {
+        "id": "Observation.extension",
+        "path": "Observation.extension"
+      },
+      {
+        "id": "Observation.extension:age-at-event",
```

Comment:
Similar to comment on `kfdrc-biospecimen.json`. I don't think this is needed since its in the profile which this one extends.

Reply:
Updated and Validated!